python/ray/tests/test_memstat.py

```diff
@@ -132,15 +132,16 @@ def make_actor():
     x_id = actor.f.remote(np.zeros(100000))
     info = ray.get(x_id)
     print(info)
-    assert num_objects(info) == 4, info
+    # Note, the actor will always hold a handle to the actor itself.
+    assert num_objects(info) == 5, info
```

Comment:
Hmm does this work properly with reference counting (not introducing a circular ref?)

It seems weird the handle is present even without calling get.